source/features/cross-deleted-pr-branches.tsx

```diff
@@ -16,20 +16,19 @@ function init(): void | false {
 		return false;
```

Comment:
I think that if you use `lastBranchAction?` we can drop the previous condition

Reply:
We need the text of the branch later on

Reply:
That's not what I suggested, no variable would be changed. It's just that `?.` would replace the `if`

Reply:
~Not sure what you mean here.~ Ohhh I see how misunderstanding. `lastBranchAction.textcontent` = the branch name that we need to delete. That is user later on in the code.

Also the code changed a bit now is it still relevant